Lib/test/test_sys.py

```diff
@@ -1287,7 +1287,7 @@ class OverflowSizeof(int):
             def __sizeof__(self):
                 return int(self)
         self.assertEqual(sys.getsizeof(OverflowSizeof(sys.maxsize)),
-                         sys.maxsize + self.gc_headsize)
+                         sys.maxsize + self.gc_headsize*2)
```

Comment:
This is because PyVarObjects use managed dicts now, right?

Reply:
Yes.